centaur/src/main/resources/standardTestCases/workbench_health_monitor_check/workbench_health_monitor_check.wdl

```diff
@@ -0,0 +1,17 @@
+version 1.0
+
+workflow workbench_health_monitor_check {
+    call run
+    output {
+        String out = run.out
+    }
+}
+
+task run {
+    command {
+        set -euo pipefail
+        curl http://localhost:8008/engine/v1/status | jq -crMS .
```

Comment:
😄 Does the Travis VM have `jq` installed ?

Reply:
jq is [_everywhere_](https://docs.travis-ci.com/user/build-environment-updates/2014-12-09/#additions)!!!